test/development/basic/allowed-dev-origins.test.ts

```diff
@@ -45,13 +45,19 @@ describe.each([['', '/docs']])(
           },
         })
 
+        // render 404 page to generate
+        // "/_next/static/chunks/pages/_app.js"
+        // we need this because not found static assets
+        // served as plain text 404 instead of HTML.
```

Comment:
```suggestion
        // render 404 page to generate
        // "/_next/static/chunks/pages/_app.js"
        // since we haven't built any paths by this point
        // causing this chunk to not be written to disk yet
```